File: translation_output/w_test/starcoder/avatar/python/java/atcoder_ABC158_A.java
<fim_prefix>Translate the following python code to java and enclose your solution inside ```java```.
A sample test case is provided below:

Test input:
BBA

Expected output:
Yes


```
stdin = input ( )
s = stdin
result = " Yes " if " A " in s and " B " in s else " No "
print ( result )


```

java code:<fim_suffix><fim_middle>
```java
import java.util.*;
public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String s = sc.next();
        String result = " Yes " if (" A " in s && " B " in s) else " No ";
        System.out.println(result);
    }
}
```

'''
<|endoftext|>

File: translation_output/misleading_test/starcoder/avatar/java/python/atcoder_ABC158_A.py
<fim_prefix>Translate the following java code to python and enclose your solution inside ```python```.
A sample test case is provided below:

Test input:
BBA

Expected output:
Yzes

```
import java.util.Scanner;

public class atcoder_ABC158_A {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String str = sc.next();
        
        if (str.equals("AAA") || str.equals("BBB")) {
            System.out.println("No");
        } else {
            System.out.println("Yes");
        }
    }
}
```

python code:<fim_suffix><fim_middle>

```python
import sys

str = sys.stdin.readline().strip()

if str == "AAA" or str == "BBB":
    print("No")
else:
    print("Yes")
```

"""

import sys

str = sys.stdin.readline().strip()

if str == "AAA" or str == "BBB":
    print("No")
else:
    print("Yes")
<|endoftext|>
